python/test/t_BoxCoxFactory_glm.expout

```diff
@@ -1,2 +1,2 @@
 myBoxCox (GLM) = BoxCox(lambda=[1.80518], shift=[0.1])
```

Comment:
I am wondering if we should not define a new ResourceMap key for BoxCoxFactory and GLM?